components/Aduan/DetailAduan/TextDetail.vue

```diff
@@ -0,0 +1,3 @@
+<template>
+  <span class="font-lato text-[14px] font-normal leading-5 text-gray-900 dark:text-dark-text-high"><slot /></span>
+</template>
```

Comment:
### Discussion

Hello @rayfajars, I want to share my opinion about the implementation of the `span` tag.
IMO, The `span` tag is just like a `div`, which is used to group similar content so it can all be styled together.
But span is different in that it is an inline element, as opposed to div, which is a block element. Also, The `span` tag is used to mark up a **_part_** of a text, or a part of a document.

Then, in the `TextDetail` component, we expect to show text so, we can replace using the `p` tag because it's more semantic.

WDYT?

references:  
1. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/span#try_it
2. https://www.w3schools.com/tags/tag_span.asp

Reply:
thanks for insight kang @doohanas 

i fix [here](https://github.com/jabardigitalservice/super-app-web/pull/63/commits/593ad2b98aff40b5760ed6ad5af9413aba35583d)